### PR TITLE
added SonataAdminBundle

### PR DESCRIPTION
**Pull request by mushkin-v**
The base options install of SonataAdminBundle

### CODE REVIEW

File: app/config/routing.yml [hunk 1]
```diff
@@ -1,3 +1,13 @@
 NelmioApiDocBundle:
     resource: "@NelmioApiDocBundle/Resources/config/routing.yml"
     prefix:   /
+
+admin:
+    resource: '@SonataAdminBundle/Resources/config/routing/sonata_admin.xml'
+    prefix: /admin
```
Comment (spolischook) [inline app/config/routing.yml]: move "admin" to parameters

File: app/config/routing.yml [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+_sonata_admin:
+    resource: .
+    type: sonata_admin
+    prefix: /admin
```
Comment (spolischook) [inline app/config/routing.yml]: move "admin" to parameters as "admin_dashboard_base_url"

**Comment by monobook** [comment]
i guess it would be nice to separate all sonata part to each one yml. in future sonata's config will be more huge and it to difficult to read one big config file.

**Comment by spolischook** [comment]
Fix code style please

**Pull request merged**